graph_def_editor/visualization/graphviz_wrapper.py

```diff
@@ -0,0 +1,375 @@
+"""Functionality to convert graph_def_editor graph to GraphViz visualization."""
+
+import re
+import uuid
+import graphviz as gv
```

Comment:
If you move this import inside the `add_digraph()` function below, you should be able to remove the hard dependency on `graphviz`.